src/yadism/esf/tmc.py

```diff
@@ -282,6 +296,50 @@ def _g2(self):
         # as we get a 1/z by the measure and an evaluation of 1-xi/z
         return self._convolute_FX("F2", g2_ker)
 
+    def _k1(self):
```

Comment:
since this is g1 specific you can also implement down there rather than in this base class